kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_componentwise.h

```diff
@@ -235,7 +235,7 @@ class ResidualBasedEliminationBuilderAndSolverComponentwise
                     EquationId[i] = geom[i].GetDof(rVar,pos).EquationId();
```

Comment:
also, to get the same behaviour we need to consistently define USE_LOCKS_IN_ASSEMBLY

Reply:
As I said, I am being consistent with the block_builder_and_solver. Maybe that should be changed there too